app/pages/Auction/BidActionPanel/BidNow.js

```diff
@@ -28,6 +30,8 @@ class BidNow extends Component {
     ownHighestBid: PropTypes.object.isRequired,
     isPending: PropTypes.bool.isRequired,
     confirmedBalance: PropTypes.number.isRequired,
+    getNameInfo: PropTypes.func.isRequired,
```

Comment:
Assuming we don't refactor the logic in `sendBid` into the wallet service, it'd be cleaner if `getNameInfo` et. al. were all passed in via `connect`. I see that `BidActionPanel` is receiving `getNameInfo` via view props, which seems tough from a maintainability standpoint given that every instance of `BidActionPanel` will need `getNameInfo`.

Reply:
Ah good point here too, will look into it.

Reply:
Wow ok sorry about this mess. Got confused by the redux structure. Fixed in e84f7de0731f4cf3b171ab9032cee23092f5081b and also fixed the same thing I did in https://github.com/kyokan/bob-wallet/pull/116